README.md

```diff
@@ -45,7 +45,7 @@ The Mixed Reality Toolkit for Unity includes many APIs to accelerate the develop
 | To develop apps for mixed reality headsets, you need the Windows 10 Fall Creators Update | The Unity 3D engine provides support for building mixed reality projects in Windows 10 | Visual Studio is used for code editing, deploying and building UWP app packages | The Emulators allow you test your app without the device in a simulated environment |
 
 # Getting started with MRTK
-To get up and going as quickly as possible, here are some guides to help you get started building Mixed Reality Projects
+To get up and startup as quickly as possible, here are some guides to help you get started building Mixed Reality Projects
```

Comment:
This isn't a proper use of the word "startup". Used as a noun, it is a newly formed company or the initial phase of a process. There is no verb form; it would be properly replaced with the word "started".

This sentence is probably too verbose in both versions. "To get started as quickly as possible" is sufficient.